surefire-providers/surefire-junit-platform/pom.xml

```diff
@@ -80,6 +80,13 @@
     <tag>surefire-3.0.0-M8</tag>
   </scm>
 
+  <properties>
+    <!-- common version suffix by platform and other junit5 artifacts, compare with https://github.com/junit-team/junit5/blob/main/gradle.properties -->
```

Comment:
@marcphilipp Is the assumption right that they always share the same suffix?

Reply:
The BOM will affect transitive dependencies of overridden plugin dependencies, and therefore leads to incompatible version.